readthedocs/core/management/commands/import_intersphinx.py

```diff
@@ -1,11 +0,0 @@
-from django.core.management.base import BaseCommand
-
-from readthedocs.builds.constants import LATEST
-from readthedocs.builds.models import Version
-from readthedocs.projects.tasks import update_intersphinx
-
-
-class Command(BaseCommand):
-    def handle(self, *args, **options):
-        for version in Version.objects.filter(slug=LATEST):
-            update_intersphinx(version.pk)
```

Comment:
Per b9a7eb668cda98c9ca1d0d2c27f96e892b7608cb and 8adb9837295b861c3838338ba0b7b30aa9019156, these both rely on code that was deleted in 2014 (see the commit message, I'll add it to the PR description too), so they don't even import without throwing an error any more.

autodoc was trying to import them and throwing warnings.